content/en/_index.md

```diff
@@ -5,13 +5,13 @@ draft: false
 ---
 
 ## Welcome to xHain
-**Hack- & Makespace in the heart of Friedrichshain, tinkering room for people interested in technology + space for net politics.**
+**Hack- & Makespace in the heart of Friedrichshain, tinkering room for people interested in technology + space for digital politics.**
 
 <br clear="all">
 
 ## Open monday nights
 
-Come by, take a look and join in! **Every Monday from 6pm until late at night in Grünberger Str. 16** in Berlin.
+Come by, take a look, and join in! **Every Monday from 6pm until late at night in Grünberger Str. 16** in Berlin. For children, we offer one Open Kids Sunday a month (1pm-6pm, typically towards the end of the month). 
```

Comment:
Suggestion:

We have an open event for kids every month. It is typically held towards the end of the month and runs from 1 – 6 p.m.